doc/book/connectors/__python.rst

```diff
@@ -43,4 +43,180 @@ Also there are several community-driven Python connectors:
 
 * `asynctnt <https://github.com/igorcoding/asynctnt>`__ with asyncio support
 * `aiotarantool <https://github.com/shveenkov/aiotarantool>`__ also with asyncio support
-* `gtarantool <https://github.com/shveenkov/gtarantool>`__ with gevent support **no active maintenance**
+* `gtarantool <https://github.com/shveenkov/gtarantool>`__ with gevent support, **no active maintenance**
+
+The table below contains a feature comparison for asynctnt, gtarantool and
+tarantool-python. aiotarantool is absent there because it is quite outdated and
+unmaintained.
+
+Feature comparison
```

Comment:
I don't know why we have two docs regarding connectors in different sections (that's odd and confusing), but as result we end up with the comparison tables in different sections (see PR #2484).

Reply:
We noticed that and have filed an issue: https://github.com/tarantool/doc/issues/2591